packer/template.json

```diff
@@ -0,0 +1,76 @@
+{
+  "builders": [
+    {
+      "boot_command": [
+        "root<enter>",
+        "<wait>",
+        "ifconfig eth0 up && udhcpc -q -n -i eth0<enter>",
+        "<wait>",
+        "wget -O - http://{{ .HTTPIP }}:{{ .HTTPPort }}/init.sh | /bin/sh<enter>"
+      ],
+      "boot_wait": "30s",
+      "communicator": "ssh",
+      "disk_size": 51200,
+      "guest_additions_mode": "disable",
+      "guest_os_type": "Linux26_64",
+      "hard_drive_interface": "sata",
+      "headless": false,
+      "http_directory": "http",
+      "iso_checksum": "af766ac6221c6f5b471ca388be22df81ac6f21be37486ba6846f115d1798528a",
+      "iso_checksum_type": "sha256",
+      "iso_url": "http://wiki.alpinelinux.org/cgi-bin/dl.cgi/v3.3/releases/x86_64/alpine-3.3.3-x86_64.iso",
+      "shutdown_command": "sudo /sbin/poweroff",
+      "ssh_password": "vagrant",
+      "ssh_username": "vagrant",
+      "ssh_wait_timeout": "5m",
+      "type": "virtualbox-iso",
+      "vboxmanage": [
+        ["modifyvm", "{{.Name}}", "--memory", "512"],
```

Comment:
This is low since this is an open source project and follows Vagrant guidelines for base boxes. It can be upped for end users' needs in Vagrantfile.